common/app/model/dotcomrendering/DotcomRenderingSupportTypes.scala

```diff
@@ -139,7 +139,6 @@ object Commercial {
 case class Config(
     switches: Map[String, Boolean],
     abTests: Map[String, String],
-    commercialBundleUrl: String,
```

Comment:
This is now included in the general config: [`JavaScriptPage.scala#L72`](https://github.com/guardian/frontend/pull/24058/files#diff-46412dd1e67b2c5c89b9988570d7c970caa8ff864c349ee8bad5443de88df07fR72).